### ISSUE TITLE
Implementation of draft-ietf-openpgp-persistent-symmetric-keys

### ISSUE THREAD

**Issue by falko-strenzke** [rnpgp/rnp#2255]
In our project we considering the implementation of [draft-ietf-openpgp-persistent-symmetric-keys](https://datatracker.ietf.org/doc/draft-ietf-openpgp-persistent-symmetric-keys/) in RNP.
Currently the draft mainly addresses the re-encryption (and authentication) of existing messages with symmetric keys that are part of the users key ring for the purpose of local (or mailbox) storage. As discussed during the IETF 120 OpenPGP session, the author Daniel Huigens is considering to widen the scope of the draft to also support the usage of the persistent symmetric keys for the purpose of communication.

@ni4 @TJ-91 
Would this draft be relevant for integration in RNP?

@kaie 
Would Thunderbird care to use this feature to allow symmetric re-encryption or entirely symmetrically encrypted and authenticated e-mail communication?

**Comment by ni4** [comment]
Could you please clarify why would you need this, given that it's in quite early stage and is not supported by any major implementations yet?
I don't think we should implement all the freshly-appeared stuff unless there is a practical need for that (like, major implementations, number of generated artifacts, and so on).

**Comment by falko-strenzke** [comment]
I don't think it is currently implemented by any major implementation yet. Proton will for sure implement it, but as far as I understand specifically for re-encryption of messages.
